### PR TITLE
Clarify a note in 5.0.2 release notes

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Clarify a note in 5.0.2 release notes

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-5.0.2.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ TiDB 版本：5.0.2
     - 为 TiCDC 扫描的速度添加背压 (back pressure) 功能 [#10151](https://github.com/tikv/tikv/pull/10151)
     - 减少 TiCDC 进行初次扫描的内存使用量 [#10133](https://github.com/tikv/tikv/pull/10133)
     - 提升了悲观事务中 TiCDC Old Value 的缓存命中率 [#10089](https://github.com/tikv/tikv/pull/10089)
-    - 提升 Region 分裂的均匀性
+    - 让 Region 分裂更均匀，优化有写入热点时 Region 大小的增长速度超过分裂速度的问题 [#9785](https://github.com/tikv/tikv/issues/9785)
```
Comment (qiancai) [inline releases/release-5.0.2.md:37]:
```suggestion
    - 让 Region 分裂更均匀，解决有写入热点时 Region 大小的增长速度超过分裂速度的问题 [#9785](https://github.com/tikv/tikv/issues/9785)
```

Comment (TomShawn) [reply releases/release-5.0.2.md:37]: According to https://github.com/pingcap/docs-cn/pull/6385#discussion_r647139007, I am not sure whether it is "解决". @MyonKeminta PTAL, thanks!

Comment (MyonKeminta) [reply releases/release-5.0.2.md:37]: cc @youjiali1995 @sticnarf

Comment (youjiali1995) [reply releases/release-5.0.2.md:37]: I prefer "优化".

Comment (sticnarf) [reply releases/release-5.0.2.md:37]: Yes, "优化" is a more conservative word so it is always not wrong.

Comment (qiancai) [reply releases/release-5.0.2.md:37]: “优化 xxx 问题” seem improper. How about "缓解 xxx 问题“？

**Review by qiancai — APPROVED** [review]

**Pull request merged**